.github/workflows/e2e-ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
   e2e-fleet-test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
I would leave `fail-fast` to true, so we get faster feedback if something fails

Reply:
Oh, fail fast in a Github Action for a test that has a matrix means that it will abort the other test running at the time the first failed. If we'd like to just rerun the failed test, it'd be good not to abort the other run. In the tests I've run, rarely two tests failed, but of course, if there's an actual issue that's not related to timings, both would fail. If you still think it'd be better to have fail-fast, let me know and I'll remove the change that disables it.

Reply:
ok, that's fine then